x-pack/plugins/apm/public/components/app/service_groups/service_group_save/create_button.tsx

```diff
@@ -7,42 +7,42 @@
 import { EuiButton } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import React from 'react';
-import { ServiceGroupsTour } from '../service_groups_tour';
-import { useServiceGroupsTour } from '../use_service_groups_tour';
+// import { ServiceGroupsTour } from '../service_groups_tour';
+// import { useServiceGroupsTour } from '../use_service_groups_tour';
 
 interface Props {
   onClick: () => void;
 }
 
 export function CreateButton({ onClick }: Props) {
-  const { tourEnabled, dismissTour } = useServiceGroupsTour('createGroup');
+  // const { tourEnabled, dismissTour } = useServiceGroupsTour('createGroup');
   return (
-    <ServiceGroupsTour
-      tourEnabled={tourEnabled}
-      dismissTour={dismissTour}
-      title={i18n.translate('xpack.apm.serviceGroups.tour.createGroups.title', {
-        defaultMessage: 'Introducing service groups',
-      })}
-      content={i18n.translate(
-        'xpack.apm.serviceGroups.tour.createGroups.content',
-        {
-          defaultMessage:
-            'Group services together to build curated inventory views that remove noise and simplify investigations across services. Groups are Kibana space-specific and available for any users with appropriate access.',
-        }
-      )}
+    // <ServiceGroupsTour
```

Comment:
@ogupte could this be removed instead?

Reply:
whoops, i'll remove it while working on alerts integration